pkg/ccl/crosscluster/logical/lww_row_processor.go

```diff
@@ -608,6 +611,9 @@ func (lww *lwwQuerier) InsertRow(
 		return batchStats{}, err
 	}
 	sess := lww.ieOverrideInsert
+	if !useLowPriority.Get(&lww.settings.SV) {
+		sess.QualityOfService = nil
```

Comment:
could we  just set this to normal so it's easier to reason about?

Reply:
nil means default IE priority, which the IE picks based on a setting. If we explicitly set to normal, then reasoning about what priority an IE, absent our setting to saying to run it at low, is running at is harder IMO since it is no longer  obeying the same setting as all the other IEs. Indeed, there isn't actually a pre-defined `*QoS` for "Normal" since nobody ever sets it to that explicitly.

Reply:
makes sense. send it.